Dockerfile

```
@@ -1,9 +1,8 @@
-FROM node:4.8.3
+FROM node:10.15.3
```

also works with newer version
```suggestion
FROM node:lts-stretch
```